### ISSUE TITLE
ENV namespacing

### ISSUE THREAD

**Issue by pikesley** [theodi/shared#692]
We now have a better `.env` mechanism, and I've now uncovered a namespacing problem. For example, we have:

    AIRBRAKE_API_KEY:
    AIRBRAKE_CALENDAR_KEY:
    AIRBRAKE_CERTIFICATE_KEY:
    AIRBRAKE_DIRECTORY_KEY:
    AIRBRAKE_GIT_VIEWER_KEY:
    AIRBRAKE_SERVICES_KEY:
    CSVLINT_AIRBRAKE_KEY:
    DASHBOARDS_AIRBRAKE_KEY:
    QUIRKAFLEEG_AIRBRAKE_KEY:

There are almost certainly more. Shouldn't be too hard to fix

**Comment by pikesley** [comment]
See also `SESSION_SECRET_CERTIFICATE` -vs- `CSVLINT_SESSION_SECRET`

**Comment by Floppy** [comment]
yeah, we improved the way we did this at one point but never backported the change to pre-existing apps.

![tech debt](http://blog.castsoftware.com/wp-content/uploads/2011/03/Tech-Debt-Trend.png)
